app/javascript/dashboard/routes/dashboard/settings/profile/MessageSignature.vue

```diff
@@ -15,7 +15,7 @@
           <woot-message-editor
             id="message-signature-input"
             v-model="messageSignature"
-            class="message-editor"
+            class="message-editor signature-editor"
```

Comment:
Can we use it as an inline

```suggestion
            class="message-editor h-[10rem]"
```